src/tools/loadbal.py

```diff
@@ -0,0 +1,517 @@
+#  Copyright (C) 2017(Dr. Horacio Andres Vargas Guzman)
+#      Max Planck Institute for Polymer Research
+#  
+#  This file is part of espressopp++ -> Powered by HeSpaDDA algorithm developed by horacio.v.g@gmail.com
```

Comment:
Same here! Why do we have to mention `Powered by HeSpaDDA algorithm developed by horacio.v.g@gmail.com` in the header?